packages/workflows/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS workflows SDK package will be documented in this file.
 
+## Recent Changes
```

Comment:
Should the changelog be updated in zosfiles package instead of the workflows package?

Reply:
Hey @gejohnston , 
I have rectified this thing, pls have a look at the updated one.